src/integration/pages/storyPage/user.amp.js

```diff
@@ -0,0 +1,7 @@
+export default () => {
+  it('I can see an amp image', () => {
```

Comment:
This seems to be repeated across the various `/integration/pages/**/user.amp.js` files - can we have it in a common file and import it where it's needed?

Reply:
@jroebu14 commented on a PR of mine for MAP integration tests - https://github.com/bbc/simorgh/pull/5880#pullrequestreview-378904399

TLDR;
I prefer to keep things explicit rather than make abstractions that don't abstract all that much. This goes against the DRY principle but I think we should "prefer duplication over the wrong abstraction" as discussed here https://www.sandimetz.com/blog/2016/1/20/the-wrong-abstraction

Reply:
@karinathomasbbc @jroebu14 Could you then explain why the analytics tests in these files are okay to be pulled out into a `common` directory, but not the user tests? 
I think I'm missing something here. 
`/integration/common/analytics.amp.js`
`/integration/common/analytics.canonical.js`

Reply:
I guess if it will work across all page types, then it would be OK to put it into the common tests - just not sure if this would work for MAPs?

Reply:
@karinathomasbbc Ah, okay, I see. So the issue is that the image isn't common to *every single* pagetype. Happy for things to stay this way in that case.